### PR TITLE
Import only needed functions from lodash

### PR DESCRIPTION
**Pull request by ddzz**
## Description
Quick PR to update all uses of lodash to import only functions that are used.

## Acceptance criteria
- [ ] CI passes.

### CODE REVIEW

File: src/applications/claims-status/utils/appeals-v2-helpers.jsx [hunk 1]
```diff
@@ -258,9 +258,9 @@ export function addStatusToIssues(issues) {
  * @returns {object} One appeal object or undefined if not found in the array
  */
 export function isolateAppeal(state, id) {
-  return _.find(
+  return find(
```
Comment (jerekshoe) [inline src/applications/claims-status/utils/appeals-v2-helpers.jsx:261]: is `find` missing an import?

Comment (ddzz) [reply src/applications/claims-status/utils/appeals-v2-helpers.jsx:261]: Yup, looks like I missed this when I was grepping.

**Review by jerekshoe — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/config/4192/index.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import _ from 'lodash';
+import { get } from 'lodash';
```
Comment (Mottie) [inline src/applications/disability-benefits/all-claims/config/4192/index.js:1]: We already have a `get` function in the platform utilities.

**Review by Mottie — APPROVED** [review]

**Review by tmitchellgcio — APPROVED** [review]

**Review by craigwheeler — APPROVED** [review]

**Review by adamwhitlock1 — APPROVED** [review]

**Review by theodur — COMMENTED** [review]
Even though we use the [babel-lodash-plugin](https://github.com/lodash/babel-plugin-lodash) and this won't decrease the bundle size, this does clean things up in the repo.

It might be a little more work, but I think it'd be cleaner if we separated this into PRs based on apps, since it's a lot of changes at once.

Also, can we add a rule to the eslint config to prevent full lodash imports from reappearing?

**Comment by ddzz** [comment]
> Even though we use the [babel-lodash-plugin](https://github.com/lodash/babel-plugin-lodash) and this won't decrease the bundle size, this does clean things up in the repo.
> 
> It might be a little more work, but I think it'd be cleaner if we separated this into PRs based on apps, since it's a lot of changes at once.
> 
> Also, can we add a rule to the eslint config to prevent full lodash imports from reappearing?

👍 Makes sense, thanks for reviewing. Yeah, I noticed this didn't decrease the bundle size. `eslint-plugin-lodash` has a rule for this, so we might want to consider adding that.

**Pull request closed without merging**